state.go

```diff
@@ -13,4 +13,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Annotations holds runtime-specified information that is not structured by this specification.
```

Comment:
On Wed, Dec 16, 2015 at 10:36:16AM -0800, Vish Kannan wrote:
> > +	// Annotations holds runtime-specified information that is not structured by this specification.
> 
> Can we rename `runtime` to `users` instead?

This *is* runtime-specified information.  User information just goes
somewhere in the bundle directory [1].

[1]: https://github.com/opencontainers/specs/pull/188#issuecomment-147026995